server/event_logging/event.py

```diff
@@ -0,0 +1,120 @@
+import json
+from enum import Enum
+from app_config import db
+
+from models import LoggedEvent
+
+
+class Event:
+    """
+    Represents an event relevant during an execution of the simulation.
+    This includes events like the start and end of an execution, as well as actions performed by players.
+    """
+
+    class Type(Enum):
+        PERFORMED_ACTION = "performed_action"
+        EXECUTION_STARTED = "execution_started"
+        EXECUTION_FINISHED = "execution_finished"
+        LOCATION_TAKE_FROM = "location_take_from"
+        LOCATION_LEAVE = "location_leave"
+        PATIENT_ARRIVE = "patient_arrive"
+        PATIENT_LEAVE = "patient_leave"
```

Comment:
I think it is not necessary to differentiate between patients and locations, as each patient is a location by definition. So I would just log `LOCATION_ARRIVE`,  `LOCATION_TAKE_FROM`, `LOCATION_LEAVE`

Reply:
FYI: In the current form there are two different implementations for leaving a location and leaving a patient, but as they have the same underlying logic they really should be treated equally. I'll write an issue to unify both processes.